client/actions/multiSelect.js

```diff
@@ -9,19 +9,37 @@ import {getItemType, gettext, planningUtils, eventUtils, getItemInArrayById} fro
 /**
  * Action Dispatcher to select an/all Event(s)
  */
-const selectEvents = (eventId, all = false) => (
+const selectEvents = (eventId, all = false, multi = false, name = '') => (
     (dispatch, getState) => {
         if (all) {
             return dispatch({
                 type: MULTISELECT.ACTIONS.SELECT_ALL_EVENTS,
                 payload: selectors.events.eventIdsInList(getState()),
             });
-        } else {
+        }
+        if (multi) {
+            const selectedIds = getState().multiSelect.selectedEventIds;
+            const prevSelectedId = selectedIds[selectedIds.length - 1];
+            const prevSelectedDate = getState().multiSelect.lastSelectedEventDate;
```

Comment:
Create two selectors for `selectedEventIds` and `lastSelectedEventDate` in `client/selectors/multiSelect.js` and use them here.